images/dind/release.sh

```diff
@@ -1,19 +1,44 @@
-set -ex
+#!/bin/bash
 
-# ./build.sh
```

Comment:
Can we keep this in there?  All of these scripts are intended to be able to run and do their entire thing (ie: release.sh requires build.sh).

Reply:
That was already commented out :-)

The point of this is to release automatically; we shouldn't need to run this manually at all - but if we do, `build.sh && release.sh` doesn't seem to hurt.

As it is, I'd rather that CI built things once, tested those then released the same artifacts. If you want a manual release.sh for use in extremis I don't mind having it call build.sh and (say) push-release.sh or something; ideally a flag with a longer character count than the characters in `build.sh &&` :-)

I'm a (soft, not dogmatic) -1 on the idea of rebuilding anything we've just tested in order to make a release.